apps/website/contents/behavioral-interview.md

```diff
@@ -22,6 +22,8 @@ sidebar_label: Step-by-step how to prepare
 
 import InDocAd from './\_components/InDocAd';
 
+_By [Lior Neu-ner](https://www.linkedin.com/in/liornn/) — former Meta engineering manager and founder of [Remote Rocketship](http://remoterocketship.com/), a job board for remote tech jobs._
```

Comment:
I added this section at the top in order to add a back link to Remote Rocketship. However, it does make it look like this entire doc was written by me, which is untrue. Do you have a suggestion on how we can add the backlink?